docs/UserGuide.md

```diff
@@ -159,6 +162,25 @@ Examples:
 * `update t/patient i/P123 n/John-Doe a/30 g/M`
 * `update t/patient i/P003 n/Tom-Doe a/20 g/M h/167 w/61 b/AB c/12341234 do/diabetes`
 
+
+### Viewing the list of patients `list -p -a -g -h -w -b -c`
```

Comment:
We are following the current convention (for example t/patient i/ID etc), maybe you can try to take a look at how to write the optional parameters such as what does [PARAM] means, [PARAM]... means, etc